### DISCUSSION TITLE
Subnet creation in VCN module

### DISCUSSION THREAD

**Discussion by karthicgit-zz** [oracle-terraform-modules/terraform-oci-vcn#90] [Q&A]
May i know any plans to add subnet creation as part of VCN module or separate module is planned for subnet creation.

**Comment by snafuz** [comment]
The plan is to implement subnet creation as a submodule of VCN module.

**Reply by djpirra** [reply]
Is this done? How do I create subnets using the module?

**Reply by snafuz** [reply]
yes, it's part of the latest release (v3.5.0). You can find an example of the input in the [terraform.tfvars.example](https://github.com/oracle-terraform-modules/terraform-oci-vcn/blob/3daf0685b56f59d6d4da856f38befda6e49bf0e8/terraform.tfvars.example#L102).

@karthicgit: could you also add a complete example [here](https://github.com/oracle-terraform-modules/terraform-oci-vcn/tree/main/examples)?
